code/modules/mob/skills/skillset.dm

```diff
@@ -78,7 +78,7 @@
 //Skill-related mob helper procs
 
 /mob/proc/get_skill_value(skill_path)
-	return skillset.get_value(skill_path)
+	return skillset?.get_value(skill_path)
```

Comment:
I've resisted doing this, as mobs without skillsets were not allowed.

Reply:
Mice had no skillsets during testing, causing a runtime here.